docs/ru/operations/server-configuration-parameters/settings.md

```diff
@@ -411,6 +411,8 @@ ClickHouse проверяет условия для `min_part_size` и `min_part
 ## interserver_listen_host {#interserver-listen-host}
 
 Ограничение по хостам, для обмена между серверами ClickHouse.
+Если используется Keeper, то такое же ограничение будет применяться к обмену данными
+между различными экземплярами Keeper.
```

Comment:
@devcrafter please check this (Deepl translation)

Reply:
@antonio2368 everything looks good ;) DeepL good enugh